tonic/Cargo.toml

```diff
@@ -74,6 +75,8 @@ tracing-futures = { version = "0.2", optional = true }
 # rustls
 tokio-rustls = { version = "0.12", optional = true }
 rustls-native-certs = { version = "0.1", optional = true }
+rustls = { version = "^0.16", features = ["dangerous_configuration"], optional = true }
```

Comment:
Do we need the `^` here?

Reply:
My bad, I thought that `0.16` meant `[0.16 - 0.17[` while `^0.16` was `>= 0.16`

Reply:
Honestly, I don't remember mostly just want to keep them similar to the others :smile: